### ISSUE TITLE
Accessibility - Full width content should have a max char count or max width

### ISSUE THREAD

**Issue by janefrommaine** [hlxsites/elixirsolutions#121]
Full width content should have a max char count or a max width so it is easier to read.

The Web Accessibility Initiative (WCAG) guideline [1.4.8](https://www.w3.org/WAI/WCAG21/Understanding/visual-presentation.html) states that, in order to be accessible to all users, lines of text should be 80 or fewer characters (or 40 or fewer characters if the text is Chinese, Japanese, or Korean).

Examples of pages with full width content:
https://main--elixirsolutions--hlxsites.hlx.live/aboutus
https://main--elixirsolutions--hlxsites.hlx.live/privacy-notice
https://main--elixirsolutions--hlxsites.hlx.live/terms-of-use
https://main--elixirsolutions--hlxsites.hlx.live/privacy-notice
https://main--elixirsolutions--hlxsites.hlx.live/non-discrimination-policy

**Comment by sforschen** [comment]
For most of the site, the future state pages would not have full-width text. the goal would be to have max 2 columns span out of 3 for text.

If we want to be closer without changing the font sizes we could move the max width from 1440px to 1280px. 

Even at 1280px 2-column span is too big (120 char) to be at 80 chars limit.

We could also consider upping the default font size from 16 to 18.  A 2 column span out of 3 plus this would be closer (120 char) at the 1440px max width. This would be the easiest fix without reducing the max width.

The closest would be combining the max width to 1280px and upping the paragraph font size to 18px (104 char) and a rule that all paragraph text cannot be full width unless it is a headline.  

Screenshot of all three solutions combined: 1280px width paragraph text size 18px and 2:3 columns.

![image](https://github.com/hlxsites/elixirsolutions/assets/134433004/eed2858e-a569-4d86-b77a-28fe211f44cc)

**Comment by sforschen** [comment]
@janefrommaine at 2 columns wide, 18px text, 1440 wide we get 84 char wide but I dont think this will work for our content

![image](https://github.com/hlxsites/elixirsolutions/assets/134433004/b901927e-d999-4c5a-892d-c5ecebc45a86)
